requirements.txt

```diff
@@ -1,3 +1,4 @@
 qiskit>=0.44
 scipy>=1.4
 numpy>=1.17
+scikit-quant
```

Comment:
The scikit-quant package is intended as an optional install https://qiskit-community.github.io/qiskit-algorithms/getting_started.html#optional-installs - it came under a varied set of licenses. This will install it for everyone which is not wanted - so this addition needs to go. The package has not been updated in ages and there is an issue open around its removal - tests with that are even skipped for some of them based on numpy as it has never been updated and CI runs on newer numpy.
* #84

Reply:
Thanks for explaining, I've now removed it from `requirements.txt`. I had erroneously added it after seeing the skipped tests.